core/templates/dev/head/pages/exploration_player/learner_local_nav.html

```diff
@@ -11,8 +11,10 @@
       <i class="material-icons">&#xE254;</i>
       <span class="oppia-icon-accessibility-label"><['I18N_PLAYER_EDIT_TOOLTIP' | translate]></span>
     </a>
+  </li>
+  <li>
     <a ng-if="!canEdit && username" class="protractor-test-exploration-suggestion-popup-link" ng-click="showLearnerSuggestionModal()" uib-tooltip="Suggest Changes" tooltip-placement="bottom" target="_blank">
-      <i class="material-icons">&#xE254;</i>
+      <i class="material-icons">contact_support</i>
```

Comment:
Please could you explain why this change was needed? It breaks convention with the other uses of `<i class="material-icons">` in the app.

If it's not needed, please revert it. Thanks!

Reply:
This change was needed because earlier 2 same icons appeared. The one that is now there for "edit" was used twice. Hence the change.

Reply:
That seems incorrect to me. I didn't think this PR was supposed to introduce a new icon altogether.

However, I'll defer to @vojtechjelinek for comment.

Reply:
@seanlip Even I thought so. But then in my last commit, @vojtechjelinek recommended me to do this as well.

Reply:
https://github.com/oppia/oppia/issues/5814#issue-376549738
@seanlip The title and description of the issue clearly stated that only one edit icon should be there.
In my first commit, I mistakenly removed the icon and didn't even replace it with anything. But then @vojtechjelinek made me realise that this icon/button had altogether a different role. It called a different function. So I ultimately replaced the original edit icon being used for "suggest change" with the most appropriate replacement.